src/nvim/option.h

```diff
@@ -60,13 +60,7 @@ typedef struct {
   /// cmdline. Only useful for string options.
   opt_expand_cb_T opt_expand_cb;
 
-  // TODO(famiu): Use OptVal for def_val.
-  union {
-    int boolean;
-    OptInt number;
-    char *string;
-  } def_val;         ///< default value for variable
-
+  OptVal def_val;     ///< default values for variable (neovim!!)
```

Comment:
- what is the "neovim!!" indicating here? 
- is "values" plural intentionally?

```suggestion
  OptVal def_val;     ///< default value
```